client/apps/edit/components/content/sections/header/controls/VideoControls.jsx

```diff
@@ -28,12 +31,12 @@ export default class VideoControls extends Component {
               data-editing
               data-type='video'
             >
-              <SectionVideo
-                editing
-                hidePreview
-                article={article}
-                section={article.heroSection}
+              <Controls
+                section={new Backbone.Model(article.get('hero_section'))}
```

Comment:
Hm should this be a proper model? Or could we use the Section model here?

Reply:
Have been trying to get rid of backbone in the header, but the video controls is the last holdout because it shares UI with the video section.  This could easily be refactored out later.